stream-chat-android-ui-components/src/main/res/values/attrs_message_list_header_view.xml

```diff
@@ -6,6 +6,8 @@
 
         <attr name="streamUiMessageListHeaderShowUserAvatar" format="boolean" />
 
+        <attr name="streamUiMessageListHeaderSeparatorBackgroundColor" format="reference|color" />
```

Comment:
Could we try using exposing `Drawable` here? Like we do it for the message composer:

```
val dividerBackground = a.getDrawable(
    R.styleable.MessageInputView_streamUiMessageInputDividerBackgroundDrawable
) ?: context.getDrawableCompat(R.drawable.stream_ui_divider)!!
```

Reply:
In this case i think color is a better approach. The view in itself is 1dp in height and i don't see advantage in exposing a drawable here. If a drawable was being used it would be just a colored drawable as well.

Reply:
Unfortunately, we don't have consistency among Views. I think we allow to configure separator Drawables for message list items and message composer. We don't provide a way to configure separator for channel list header.

But I'm fine with separator color as well as it solves the problem 👍

Reply:
For consistency sake i can refactor it to be a drawable. Also i can add the same capability to the channel list header so that similar components behave the same.

Reply:
It's up to you. I will check your solution with colors and approve.